azafea/event_processors/endless/activation/v1/handler.py

```diff
@@ -52,6 +51,8 @@ class Activation(Base):
 
     __table_args__ = (
         CheckConstraint('char_length(country) = 2', name='country_code_2_chars'),
+        CheckConstraint('latitude = FLOOR(latitude) +0.5', name='latitude_floor'),
+        CheckConstraint('longitude = FLOOR(longitude) +0.5', name='longitude_floor'),
```

Comment:
Perhaps there could be a better name for these constraints. "latitude_precision_reduced"? "latitude_truncated"?

How about verifying they are in the correct range?

Reply:
And let's be consistent in the case of `FLOOR` and `floor` between here and the migration. Lower-case seems conventional according to https://www.postgresql.org/docs/current/functions-math.html:

```suggestion
        CheckConstraint('latitude = floor(latitude) + 0.5', name='latitude_precision_reduced'),
        CheckConstraint('longitude = floor(longitude) + 0.5', name='longitude_precision_reduced'),
```